packages/material-ui-lab/src/internal/pickers/PickersPopper.tsx

```diff
@@ -87,27 +86,37 @@ const PickersPopper: React.FC<PickerPopperProps & WithStyles<typeof styles>> = (
 
     if (open) {
       lastFocusedElementRef.current = document.activeElement;
+
+      // Focus the paper element when popper opens, otherwise onBlur will not be invoked.
+      // Remove "disableAutoFocus" prop on TrapFocus also works but TrapFocus seems buggy.
```

Comment:
Could you describe the bug? Preferably with a test.

Reply:
After Removing "disableAutoFocus" prop and the code `paperRef.current?.focus();` that manually focus the paper component, *TrapFocus* focuses the paper component and shift the page up when open the picker that located in the middle to bottom of the screen.